content/news/index.cs.md

```diff
@@ -0,0 +1,9 @@
+---
```

Comment:
Please rename this file to _index.cs.md

It's easy to test it locally by running `zola serve`